runtime/port/unix/j9sysinfo.c

```diff
@@ -257,6 +257,14 @@ static intptr_t getAIXPPCDescription(struct J9PortLibrary *portLibrary, J9Proces
 #define __power_10() (_system_configuration.implementation == POWER_10)
 #endif /* !defined(__power_10) */
 
+/*
+ * Please update the macro below to stay in sync with the latest POWER processor known to OpenJ9,
+ * ensuring CPU recognition is more robust than in the past. As the macro currently stands, any
+ * later processors are recognized as at least POWER11.
+ */
+#define POWER11_OR_ABOVE (0xFFFFFFFF << 19)
```

Comment:
Suggest this would be clearer if the number `11` were present, i.e.
```c
#define POWER11_OR_ABOVE (0xFFFFFF00 << 11)
```

Reply:
how is that clearer? looks like cutting feet to fit with the shoes ...
originally, i had POWER_11 definition there (0x00080000) to make the derivation clear.

Reply:
So far, recognizing POWERN, where 8 <= N <= 10, uses the mask `(0x100 << N)`: I'm just suggesting that pattern should be more visible.